common/determined_common/api/experiment.py

```diff
@@ -246,7 +246,7 @@ def generate_random_value(hparam: Any) -> Any:
                 return math.pow(hparam["base"], random.uniform(hparam["minval"], hparam["maxval"]))
             else:
                 raise Exception("Wrong type of hyperparameter: {}".format(hparam["type"]))
-        elif isinstance(hparam, (int, float, str)):
```

Comment:
q: why do we need to support `None`? It wouldn't fit with the cluster mode because `None` can't be serialized.

Reply:
Oops, forgot to respond.

Yes, we totally support this master-side.  None is definitely serializable; it becomes a literal `null` in json.